api/data_refinery_api/urls.py

```diff
@@ -78,66 +78,145 @@ class AccessUser:
-        path('transcriptome_indices/', include([
-            path('', TranscriptomeIndexList.as_view(), name='transcriptome_indices'),
-            path('<int:id>', TranscriptomeIndexDetail.as_view(), name='transcriptome_indices_read'),
-        ])),
-
-        # QN Targets
-        url(r'^qn_targets/$', QNTargetsAvailable.as_view(), name='qn_targets_available'),
-        url(r'^qn_targets/(?P<organism_name>.+)$', QNTargetsDetail.as_view(), name='qn_targets'),
-
-        # Computed Files
-        url(r'^computed_files/$', ComputedFilesList.as_view(), name='computed_files'),
-        url(r'^original_files/$', OriginalFileList.as_view(), name='original_files'),
-        url(r'^computational_results/$', ComputationalResultsList.as_view(), name='results'),
-
-        # Compendia
-        url(r'^compendia/$', CompendiumResultList.as_view(), name='compendium_results'),
-        url(r'^compendia/(?P<id>[0-9]+)/$', CompendiumResultDetails.as_view(), name='compendium_result'),
-
-        # v1 api docs
-        url(r'^swagger/$', schema_view.with_ui('swagger', cache_timeout=0), name='schema_swagger_ui'),
-        url(r'^$', schema_view.with_ui('redoc', cache_timeout=0), name='schema_redoc'),
-    ])),
-
+    url(
+        r"^(?P<version>v1)/",
+        include(
+            [
+                # Primary search and filter interface
+                url(
+                    r"^search/$",
+                    ExperimentDocumentView.as_view({"get": "list"}),
+                    name="search",
+                ),
+                url(r"^experiments/$", ExperimentList.as_view(), name="experiments"),
+                url(
+                    r"^experiments/(?P<accession_code>.+)/$",
+                    ExperimentDetail.as_view(),
+                    name="experiments_detail",
+                ),
+                url(r"^samples/$", SampleList.as_view(), name="samples"),
+                url(
+                    r"^samples/(?P<accession_code>.+)/$",
+                    SampleDetail.as_view(),
+                    name="samples_detail",
+                ),
+                url(r"^organisms/$", OrganismList.as_view(), name="organisms"),
+                url(r"^platforms/$", PlatformList.as_view(), name="platforms"),
+                url(r"^institutions/$", InstitutionList.as_view(), name="institutions"),
+                url(r"^processors/$", ProcessorList.as_view(), name="processors"),
+                # Deliverables
+                url(r"^dataset/$", CreateDatasetView.as_view(), name="create_dataset"),
+                url(
+                    r"^dataset/(?P<id>[0-9a-f-]+)/$",
+                    DatasetView.as_view(),
+                    name="dataset",
+                ),
+                url(r"^token/$", CreateApiTokenView.as_view(), name="token"),
+                url(
+                    r"^token/(?P<id>[0-9a-f-]+)/$",
+                    APITokenView.as_view(),
+                    name="token_id",
+                ),
+                # Jobs
```

Comment:
Note - these used to look pretty consistent but now the comments / line breaks are not as clear & the format of each URL differs by line length.

Reply:
yes, it breaks any line above `88` characters. We could increase that limit to allow for longer lines.

Reply:
I'm not sure that we want to change the line length to accomplish that as the issue will just occur somewhere else at a different threshold. I'm not opposed to this formatting - just noting the only caveat I saw if we went this way. It helped me notice #2023 so that's not all bad!